### PR TITLE
Translation fixes for help-resources.php

### PR DESCRIPTION
**Pull request by NikV**
https://github.com/pods-framework/pods/issues/2850. Quick question: do we want to translate the name of the recommended plugins? Also, random hook doc!

### CODE REVIEW

**Comment by Shelob9** [comment]
Plugin names are not translation needed.

**Comment by NikV** [comment]
Plugin Descriptions don't need it either?

File: ui/admin/help-resources.php [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+	<li><?php _e('To learn more about using Pods, see the', 'pods'); ?><a href="http://pods.io/docs/" target="_blank">documentation</a>, or our <a href="http://pods.io/tutorials/" target="_blank">tutorials section</a>.</li>
 
-	<li>To get help with a specific issue, you can ask in our <a href="http://pods.io/forums/" target="_blank">support forums</a>, or in <a href="http://pods.io/forums/chat/" target="_blank">our chat</a>.
+	<li><?php _e('To get help with a specific issue, you can ask in our <a href="http://pods.io/forums/" target="_blank">support forums</a>, or in <a href="http://pods.io/forums/chat/" target="_blank">our chat</a>.', 'pods'); ?></li>
 
-	<li>To report <strong>bugs or request features</strong>, go to our <a href="https://github.com/pods-framework/pods/issues?sort=updated&direction=desc&state=open" target="_blank">GitHub</a>.</li>
+	<li><?php _e('To report <strong>bugs or request features</strong>, go to our <a href="https://github.com/pods-framework/pods/issues?sort=updated&direction=desc&state=open" target="_blank">GitHub</a>.', 'pods');  ?></li>
```
Comment (Shelob9) [inline ui/admin/help-resources.php]: @NikV Don't put HTML in the translation string. See: http://ottopress.com/2012/internationalization-youre-probably-doing-it-wrong/ Heading "Annoyance the First: Thou shalt not put unnecessary HTML markup into the translated string." 

I have 100% adopted Otto's opinions on this topic that are expressed in that article. Please assimilate said knowledge.

**Comment by NikV** [comment]
Oh doh, sprintf! Will change soon

**Pull request closed without merging**